edx_sga/static/js/src/edx_sga.js

```diff
@@ -376,8 +381,47 @@ function StaffGradedAssignmentXBlock(runtime, element) {
                   );
                 });
             }
+            $("#lean_overlay").on("click", function (_){
+            if (currentIFrameHeight && !gradePopUpIsOpen) {
+              sendResizeMessage(currentIFrameHeight)
+              currentIFrameHeight = null
+            }
+            })
         });
 
+        function updateIframe(){
+          if(isInIframe && !currentIFrameHeight){
+            currentIFrameHeight = $("body").height()
+            addMaxHeightInIframe()
+            if(currentIFrameHeight < 600){
+              sendResizeMessage(currentIFrameHeight + 600)
```

Comment:
I am thinking what if the height of the body is already closed to 600 but not exactly 600 so adding another 600 would not be ideal. In fact, that would add another scroll and the iframe pop size would again become more than the screen size. What's your opinion on that. Could you try to check the behavior of that manually in inspect mode?

Reply:
This wont be a problem  since we are also adding a max-height constraint to fix the popup blowing up incase the iframe itself is too large

Reply:
then why are we adding 600 to currentIframeheight? Instead, we should set it to 600.